app/src/main/java/org/literacyapp/authentication/helper/DetectionHelper.java

```diff
@@ -126,18 +128,15 @@ public static synchronized void adjustScreenBrightness(Context context, Mat img)
         if (currentImageBrightness < IMAGE_BRIGHTNESS_THRESHOLD){
             setScreenBrightnessMode(context);
             setScreenBrightness(context, currentImageBrightness);
+            setDisplayTemperatureNight();
         }
     }
 
     private static synchronized void setScreenBrightnessMode(Context context){
```

Comment:
`setScreenBrightnessMode` to what? Can you add some JavaDoc to explain how this method changes the current settings?